### ISSUE TITLE
List properties split button is missing the `aria-label` attribute

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#12465]
## 📝 Provide a description of the improvement

The split button arrow for lists is missing a proper `aria-label` attribute, e.g. `Numbered List Options`. 

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by Dumluregn** [comment]
Hi, I see that arrows for other splitbuttons also don't have the `aria-label` attribute, but they do have `aria-labelledby` one. AFAIU from [docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) it makes the `aria-label` unnecessary. @mlewand could you verify that with one of our a11y experts perhaps? Because I think either this ticket is invalid or we need to address all the splitbuttons, not just this one.

**Comment by mlewand** [comment]
It does indeed have a `arial-labelledby` attribute. So @Dumluregn is right.

However I noticed that all of them have the same label as the main button and I wonder if that should be the case.

Example:

*   Numbered list down arrow is labelled as "Numbered List". Is this correct? Well in the end it does turn into a numbered list just like the main button (only different numbering).
*   Code block has "Insert code block" label. It also inserts a code block like a main button (only difference is that it has a language predefined).

Keep in mind that arrow buttons differ from face buttons in a way that they have `aria-haspopup="true"` which is announced by screen reader.

Would appending "options" word improve discoverability? Maybe, but I'm not really convinced on this 🤔 The current behavior seems to be good enough for me.

While adding a second solution we'd have to account for different languages, because not all languages will have a pattern that it's just enough to add a single word. Actually sometimes the down arrow dropdown might have different behavior than the face - so this means we'd have to complicate the API for that.

Any thoughts on that?
